Python/Product/Debugger.Concord/CppExpressionEvaluator.cs

```diff
@@ -32,7 +32,10 @@ internal class CppExpressionEvaluator {
         private readonly DkmStackWalkFrame _nativeFrame;
         private readonly DkmInspectionContext _cppInspectionContext;
 
-        public CppExpressionEvaluator(DkmInspectionContext inspectionContext, DkmStackWalkFrame stackFrame) {
+        public CppExpressionEvaluator(DkmInspectionContext inspectionContext, DkmStackWalkFrame stackFrame) 
```

Comment:
I added some overloads here that allowed me to run the EE with just a stack frame and to allow side effects (necessary in order to call a function).